tests/integration/test_pocket.py

```diff
@@ -0,0 +1,50 @@
+import responses
+from flask import Flask
+
+from archivy import data
+from archivy.extensions import get_db
+
+
+def test_parse_pocket(test_app, client, mocked_responses):
+    """Test the /pocket endpoint by mocking out the external HTTP calls to the
+    pocket API
+    """
```

Comment:
Could you also add a comment here to explain the `responses` part?

Something like this:

The `mocked_responses` are a way of declaring that when the [test client](https://werkzeug.palletsprojects.com/en/1.0.x/test/#werkzeug.test.Client) calls a GET on for example, "https://example.com", instead of actually doing the web request we set a response that will be returned at the point where the request is called.

This allows us to define a stable behaviour that is not constrained by the instability of the web.

Reply:
I added some docstrings and comments, is it clear enough now?

Reply:
Oh you've done it above :smile:

Reply:
Yes it's perfect.